dali/kernels/signal/decibel/decibel_calculator.h

```diff
@@ -25,6 +25,7 @@ namespace signal {
 template <typename T>
 struct DecibelCalculator {
  public:
+  DALI_HOST_DEV DALI_FORCEINLINE
   explicit DecibelCalculator(T mul = 10.0, T s_ref = 1.0, T min_ratio = 1e-8)
       : inv_mul_(1. / mul)
```

Comment:
This forces double precision calculation.

Reply:
Don't you think it's ok to have this `inv_mul_` calculated with double precision?

Reply:
done